airbyte-scheduler/scheduler-persistence/src/test/java/io/airbyte/scheduler/persistence/DefaultJobCreatorTest.java

```diff
@@ -351,13 +357,16 @@ void testCreateResetConnectionJob() throws IOException {
     final String expectedScope = STANDARD_SYNC.getConnectionId().toString();
     when(jobPersistence.enqueueJob(expectedScope, jobConfig)).thenReturn(Optional.of(JOB_ID));
 
-    final long jobId = jobCreator.createResetConnectionJob(
+    final Optional<Long> jobId = jobCreator.createResetConnectionJob(
         DESTINATION_CONNECTION,
         STANDARD_SYNC,
         DESTINATION_IMAGE_NAME,
         List.of(STANDARD_SYNC_OPERATION),
-        List.of(STREAM_DESCRIPTOR1, STREAM_DESCRIPTOR2)).orElseThrow();
-    assertEquals(JOB_ID, jobId);
+        List.of(STREAM_DESCRIPTOR1, STREAM_DESCRIPTOR2));
+
+    verify(jobPersistence).enqueueJob(expectedScope, jobConfig);
+    assertTrue(jobId.isPresent());
+    assertEquals(JOB_ID, jobId.get());
```

Comment:
I updated these tests a bit to actually verify that the JobResetConnectionConfig defined in the test is actually what is being passed to `enqueueJob`. Without these changes, the tests would still pass regardless of if I set the state or not